app/repositories/submission_repository.rb

```diff
@@ -31,6 +31,10 @@ def waitlist
     rated_scope.offset(Setting.get.available_spots).to_a
   end
 
+  def to_invite
+    rated_scope.where.not(confirmation_status: 3).limit(Setting.get.available_spots).where(confirmation_status: 0).to_a
```

Comment:
It won't work with enums. I'm gonna do this: `.where('confirmation_status <> ?', Submission.confirmation_statuses[:expired])`